app/routes/contact/__init__.py

```
@@ -14,118 +14,217 @@
+                    "email": data.email,
+                    "company": data.company,
+                    "phone": data.phone
+                },
+                geo_data=geo_data
+            )
+            if not success:
+                logger.warning("Failed to track call booking conversion in database")
+            else:
+                logger.info("Call booking analytics conversion tracked successfully in database")
+        except (ImportError, SQLAlchemyError) as e:
+            logger.exception(f"Failed to track analytics conversion: {e}")
```

Ruff (0.13.1)

Redundant exception object included in `logging.exception` call

(TRY401)